diff --git a/node_modules/contentful/dist/types/types/content-type.d.ts b/node_modules/contentful/dist/types/types/content-type.d.ts
index 795b738..748c6fc 100644
--- a/node_modules/contentful/dist/types/types/content-type.d.ts
+++ b/node_modules/contentful/dist/types/types/content-type.d.ts
@@ -61,7 +61,7 @@ export interface ContentTypeFieldValidation {
     };
     linkMimetypeGroup?: string[];
     in?: string[];
-    linkContentType?: string[];
+    linkContentType?: string | string[];
     message?: string;
     nodes?: {
         'entry-hyperlink'?: ContentTypeFieldValidation[];
